src/main/java/com/xebialabs/deployit/ci/Credential.java

```diff
@@ -33,6 +33,7 @@
 import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import javax.ws.rs.POST;
```

Comment:
Wrong annotation, you want 
```suggestion
import org.kohsuke.stapler.verb.POST;
```

Reply:
The stapler version downloaded while building the plugin is 1.237 and this stapler version does not have verb folder(hence POST.java is not available), hence package is not resolved and build fails. Alternatively, as recommended in SECURITY-1982, I am using @RequirePOST annotation.

![stapler_version](https://user-images.githubusercontent.com/55885937/120498754-1ad09c80-c3dd-11eb-84db-83ecacdcec0b.PNG)

![stapler_detail](https://user-images.githubusercontent.com/55885937/120498805-23c16e00-c3dd-11eb-8fa3-764c51f24237.PNG)

Reply:
That works too.

It's probably time to update your core dependency: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ + http://stats.jenkins.io/pluginversions/deployit-plugin.html